scripts/prepare.js

```diff
@@ -12,23 +18,23 @@ if (!fs.existsSync("analyzers")) {
 }
 
 downloadIfNeeded(
-  "https://repox.sonarsource.com/sonarsource/org/sonarsource/sonarlint/core/sonarlint-language-server/3.1.0.1376/sonarlint-language-server-3.1.0.1376.jar",
+  `https://repox.sonarsource.com/sonarsource/org/sonarsource/sonarlint/core/sonarlint-language-server/${languageServerVersion}/sonarlint-language-server-${languageServerVersion}.jar`,
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Split this 183 characters long line (which is greater than 120 authorized). ([javascript:LineLength](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3ALineLength))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarlint.vscode%3Asonarlint-vscode&pullRequest=23&issues=AWNiy45u6IBDU_gafCSS&open=AWNiy45u6IBDU_gafCSS)

[ISSUE_KEY]: AWNiy45u6IBDU_gafCSS